include/revng/ABI/Support.h

```diff
@@ -53,65 +53,19 @@ inline model::TypePath replaceAllUsesWith(const model::Type::Key &OldKey,
 ///       Pre-padding (offset) needs to be taken care of separately.
```

Comment:
What is this commit doing? Please be specific in the commit message.

Reply:
I guess if I'm confused about which one, the answer is both, isn't it? 😅

Reply:
Eh, I don't remember, you can check all the commits and see which one has "Pre-padding (offset) needs to be taken care of separately." at the top.
Next time I'll put a reference to the commit message.

Reply:
Either way, I updated both 😅. What do you think?